resolwe_bio/processes/expression/bowtie2_quant.yml

```diff
@@ -0,0 +1,137 @@
+# =======================
+# Expression from bowtie2
+# =======================
+---
+
+- slug: bowtie2-quant
```

Comment:
Why not `shrna-quant`?

Reply:
If you will change this, update also `type` accordingly.

Reply:
This is a better suggestion, thanks.

Reply:
I have renamed the yml file to `shrna_quant.yml` as well as all types and slug reference(s).